Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts

```diff
@@ -9,6 +10,8 @@ const calculateTimeUntil = (date: Date) => intervalToDuration({
 
 const formatDate = (date: Date, formatString = defaultDateTimeFormat) => format(date, formatString);
 
+const formatDateAsMomentToString = (date: Date, formatString = defaultDateTimeFormat) => moment(date).format(formatString);
```

Comment:
Isn't this name too explicit? Also, what is wrong with the above method `formatDate`?